src/lib/env.ts

```diff
@@ -38,4 +38,5 @@ export const env = {
   passThroughNonReverting: getBoolean(
     getEnvVar("PASS_THROUGH_NON_REVERTING", fallback.passThroughNonReverting.toString()),
   ),
+  flashbotsOrigin: getEnvVar("FLASHBOTS_ORIGIN", ""),
```

Comment:
why use empty string rather than undefined for tthis if not defined in env?

Reply:
fixed to use optional `env.flashbotsOrigin`